decidim-admin/app/views/layouts/decidim/admin/_sidebar_menu.html.erb

```diff
@@ -6,7 +6,16 @@
     </div>
   </div>
 
-  <div class="item_show__wrapper">
-    <%= yield %>
-  </div>
+  <% if defined?(has_tab_menu?) && has_tab_menu? %>
+    <div class="layout-content__container">
+      <%= admin_tabs(tab_menu_name).render %>
+      <div class="item_show__wrapper">
+        <%= yield %>
+      </div>
+    </div>
```

Comment:
Can we leave the white background here?

I understand that what you're proposing is better because it's possible to see the tabs clearly, and with the white background we have a lack of contrast situation.

But I also think that I prefer to have the background consistent inside a view with the sidebar, so I prefer to err in the consistency side. Mind that this problem is also visible in other places, such as some input fields. See here:

![image](https://github.com/decidim/decidim/assets/717367/667170d2-3e4d-4b64-aef6-1cbce4fb20ff)

We should create an issue to keep don't loose these contrast issues for sure. 

```suggestion
    <div class="item_show__wrapper">
      <%= admin_tabs(tab_menu_name).render %>
      <%= yield %>
    </div>
```

Reply:
This is done in 7388ad485c

Reply:
Much cleaner than my proposal! 💯